tasks/installubuntu.yml

```diff
@@ -0,0 +1,47 @@
+# roles/mariadb/tasks/install.yml
+---
+- name: Check Python3 present
```

Comment:
I'd rather not have to add code that checks Python versions, that seems outside the scope of this role.